modules/claims_api/catalog-info-v2.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: backstage.io/v1alpha1
+kind: API
+metadata:
+  name: benefits-claims-v2
+  description: The Benefits Claims API is a suite of services that enables VA developers to build applications that let Veterans view and submit certain claims and claim-related information to the VA. We provide the ability to auto-establish claims by submitting structured data that can return realtime validations, allowing for better data quality.
+  annotations:
+    github.com/project-slug: https://github.com/department-of-veterans-affairs/vets-api
+    datadog/production-monitor-id: '98a-nx5-icf'
+    datadog/sandbox-monitor-id: '2ne-mbd-mve' # status tab
```

Comment:
I'll DM my PoC once we confirm the v1 values are working to get these. Just don't want to pester them until we know we're good. 👍